week 6.cpp

```diff
@@ -0,0 +1,204 @@
+#include <iostream>
+#include <stdlib.h>
+using namespace std;
+int id=0;
+class Item
+{
+	int item_id;
+	//char item_name;
+	float price;
+	
+	
+	public:
+
+	int item_count;
```

Comment:
Should item_count be a member of the Item class? It seems like it should be a member of a vending machine or inventory class.  As a thought, a bag of chips doesn't know how many other chips there are in the machine.